IRequests/RequestSignUp.cpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include "IRequests.h"
+#include "RequestSignUp.h"
+#include <cpprest/json.h>
+#include "../stringtowstring.h"
+using namespace web;
+
+RequestSignUp::RequestSignUp(MSSQLDatabase* d,answercontainerinterface* answercontainer, const ISXModel::User& user) : IRequests(db, answercontainer), user(user) {}

```

Comment:
you are passing 'answercontainerinterface' here as parameter but I do not see usage of it in this class.